data/README.md

```diff
@@ -38,6 +38,8 @@
 | datapage_v1-snappy-compressed-checksum.parquet | compressed INT32 columns in v1 data pages with a matching CRC          |
 | datapage_v1-corrupt-checksum.parquet           | uncompressed INT32 columns in v1 data pages with a mismatching CRC     |
 | overflow_i16_page_cnt.parquet                  | row group with more than INT16_MAX pages                   |
+| bloom_filter.bin                               | deprecated bloom filter binary with binary header and murmur3 hashing |
```

Comment:
What do you mean with "binary header"? Doesn't the [murmur3 bloom filter format](https://github.com/apache/parquet-format/commit/54839ad5e04314c944fed8aa4bc6cf15e4a58698) already use a Thrift header (`BloomFilterPageHeader`)?

Reply:
Hi pitrou, I'm also confuse with this. But original code, with works well with `bloom_filter.bin`, uses binary parsing